_posts/2023-08-23-developer-ergonomics-hier-kommen-angular-signals/2023-08-23-developer-ergonomics-hier-kommen-angular-signals.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Developer Ergonomics mit Angular Signals
+description: In diesem Artikel erfährst Du, warum das Angular Team sich für Signals entschieden hat, wie Du Signals in Deiner Anwendung einsetzen kannst und was es für die Zukunft von RxJS im Kontext von Angular bedeutet.
+author: "Philipp Escher"
+published_at: 2023-08-23 15:00:00
+header_source: https://unsplash.com/de/fotos/xXiKQ2AavlY
+header_image: header.jpg
+categories: "angular signals update"
+---
+
+## Das Angular 16 Update
+
+Vielleicht hast Du es mitbekommen: Mit Angular 16 wurde das, gemessen am Umfang, größte Update nach dem initialen Release von Angular veröffentlicht. Neben Features wie Hydration oder dem Support für Typescript 5.0 enthält das Update drei Reactivity Primitives und damit einen Mechanismus für Reaktivität. Sarah Drasnger, Director of Engineering bei Google, spricht auf [Twitter bzw. X](https://twitter.com/sarah_edo/status/1628065696247857152) sogar von einer Angular Renaissance.
+
+## Motivation hinter Signals
+
+Während andere Frameworks bereits länger Mechanismen für Reaktivität anbieten, ist es bei Angular eine Neuheit. Signals sollen Angular leichtgewichtiger machen und in Zukunft auch ermöglichen, ohne Zone.js zu arbeiten. Mit Signals soll Angular fähig werden, genau zu wissen, welche Komponenten aktualisiert werden müssen.
```

Comment:
- Mechanismen für Reaktivität hat Angular schon immer -> RxJS, **nur kein Reaktives Primitiv** (built-in)
- 
```suggestion
Während andere Frameworks bereits _Reactive Primitivs_ anbieten, ist es bei Angular eine Neuheit. Signals sollen Angular leichtgewichtiger machen und in Zukunft auch ermöglichen, ohne Zone.js zu arbeiten. Mit Signals soll der Angular Change-Detection emöglicht werden fein granularer zu arbeiten und wirklich nur Neuerungen oder Änderungen zu rendern, anstatt den kompletten Compoennt-Tree überprüfen zu müssen.
```